ossapi/ossapiv2.py

```diff
@@ -195,20 +197,33 @@ def _save_token(self, token):
         with open(self.AUTHORIZATION_TOKEN_FILE, "wb+") as f:
             pickle.dump(token, f)
 
-    def _get(self, type_, url, params={}):
+    def _get(self, type_, url, params=None):
+        params = params or {}
```

Comment:
any reason you changed the default arg for `params` from `{}` to `None`?

Reply:
I've been caught out by mutable default arguments before; switching to this format is just habit for me now. We don't mutate the dict anywhere so I'm happy to leave it as it was.